### PR TITLE
Update /committee after 2022 AGM

### PR DESCRIPTION
**Pull request by AlexJBanks**

### CODE REVIEW

**Review by JTWWilson — APPROVED** [review]
I think this is ready to go. Once we have Tori's bio then we can update that and put it in. For now, I think it's best to get the rest uploaded on live.

**Comment by rainestormee** [comment]
Can we not have a blank bio for them? Or something that seems generic… “x person is holding y position for the new committee starting in z”

**Comment by LMBishop** [comment]
> Can we not have a blank bio for them? Or something that seems generic… “x person is holding y position for the new committee starting in z”

I think it's either this or just remove it entirely

File: _data/committee/2022-23.yaml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  - name: Raine Arrowsmith
+    pronouns: She/They
+    discordtag: Raine#8544
+    role: Secretary
+    picture: /assets/committee/2022-23/mini/raine.png
+    start-date: 1/7/2022
+    bio: >
+      Hello, I'm Raine and I'm going into my final year studying Computer Science. I will be responsible for all of your sports 
+      and e-Sports events this year -- please let me know what you want to see! When I'm not sleeping I'm either speedrunning,
```
Comment (JTWWilson) [inline _data/committee/2022-23.yaml:34]: @rainestormee are you able to update this?

**Review by JTWWilson — CHANGES_REQUESTED** [review]

**Comment by MattyTheHacker** [comment]
> > Can we not have a blank bio for them? Or something that seems generic… “x person is holding y position for the new committee starting in z”
> 
> I think it's either this or just remove it entirely

I think a generic one is fine until a proper one can be written

**Comment by JTWWilson** [comment]
> I think a generic one is fine until a proper one can be written

Something like: "Hi, I'm Tori and I'm a second year Computer Science student. I'm the Interim First Year Rep and I'll be helping organise events for first years joining at the start of this year. Feel free to say hi if you see me on the Discord!"

File: _data/committee/2022-23.yaml [hunk 2]
```diff
@@ -0,0 +1,151 @@
+academic_year: 2022/2023
```
Comment (rainestormee) [inline _data/committee/2022-23.yaml:1]: Going to ask tori to be added in with the bio that Jacob made

**Review by MattyTheHacker — APPROVED** [review]
Think that should be good now I think

**Review by rainestormee — APPROVED** [review]
I’m happy

**Review by LMBishop — APPROVED** [review]

**Pull request merged**